sphinx/templates/latex/tabular.tex_t

```diff
@@ -1,3 +1,10 @@
+<%- if table.align %>
+  <%- if table.align == 'center' -%>
+    \begin{center}
+  <%- elif table.align in ('left', 'right') -%>
+    \begin{flush<%= table.align%>}
+  <%- endif -%>
+<% endif -%>
```

Comment:
memo: this method adds extra vertical spaces due to nature of center, flushleft, flushright environments.